content/blog/2016/2016-08-10-rails-cd-with-pipeline.adoc

```diff
@@ -0,0 +1,358 @@
+---
+layout: post
+title: "Continuous Security for Rails apps with Pipeline and Brakeman"
+tags:
+- ruby
+- pipeline
+- rails
+- brakeman
+- continuousdelivery
+author: rtyler
+---
+
+:sectanchors:
+
+
+[NOTE]
+--
+This is a guest post by link:https;//github.com/rtyler[R. Tyler Croy], who is a
+long-time contributor to Jenkins and the primary contact for Jenkins project
+infrastructure. He is also a Jenkins Evangelist at
+link:http://cloudbees.com[CloudBees, Inc.]
+--
+
+
+When the link:http://rubyonrails.org[Ruby on Rails] framework debuted it
+changed the industry in two noteworthy ways: it created a trend of opinionated web
+application frameworks (link:https://www.djangoproject.com/[Django],
+link:https://playframework.com/[Play], link:https://grails.org/[Grails]) and it
+also _strongly_ encouraged thousands of developers to embrace test-driven
+development along with many other modern best practices (source control, dependency
+management, etc). Because Ruby, the language underneath Rails, is interpreted
```

Comment:
I don't disagree with the convoluted part, but disagree with chopping it out entirely. The point I'm trying to get across is that pretty much "out of the box" Rails apps, by convention, are ideal/ready for continuous delivery